versions/1.1.0-dev.md

```diff
@@ -252,7 +252,7 @@ info:
   title: Apply Traits
   version: 1.0.0
 actions:
-  - target: $.paths.*.get[?@.x-oai-traits.paged]
+  - target: $.paths[?@.get['x-oai-traits'][?@ == 'paged']].get
```

Comment:
[Playing with bump-cli and speakeasy](https://github.com/ralfhandl/overlay-experiments) I found that this works:

```suggestion
  - target: $.paths.*.get[?(@.x-oai-traits[?(@ == 'paged')])]
```

Both tools require the parens after the `?` although [RFC9539](https://www.rfc-editor.org/rfc/rfc9535#tbl-filter) has 25 examples without parens and just one with parens to tell that expressions with parens are equivalent to ones without 😁